.github/workflows/build-and-test.yml

```diff
@@ -12,32 +12,19 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
I couldn't comment on the hidden lines, but I think you might also be able to remove `STK_JARS_PATH` and `STK_PROPERTIES_PATH` here too.

Reply:
Yeah, both are not required. But LICENSE_DATA is required